scripts/scil_compute_todi.py

```diff
@@ -1,9 +1,11 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 """
-Compute a length-weighted Track Orientation Density Image (TODI).
-This script can afterwards output a length-weighted Track Density Image
-(TDI) or a length-weighted TODI, based on streamlines' segments.\n\n
+Compute a Track Orientation Density Image (TODI).
+Each segment of the streamlines is weighted by length,
```

Comment:
Sound weird. Maybe something like this: Each segment of the streamlines is weighted by length (to support compressed streamlines)

Reply:
Agree with this suggestion

Reply:
done